docs-ref-conceptual/azure-cli-vm-tutorial-5.md

```diff
@@ -47,7 +47,7 @@ In order to use command-line tools, change the command to remove the custom JSON
 the shell `read` command to load results into multiple variables. Since two values are displayed on separate lines, the `read` command
 delimiter must be set to the empty string rather than the default of non-newline whitespace.
 
-```bash
+```azurecli
```

Comment:
It looks like bash is assigning values returned by the CLI command to its variables. This could be another one of those ambiguous code blocks, but since Azure CLI is doing most of the heavy lifting, I think we can keep the azurecli tag.